ldms/src/ldmsd/ldmsd_request_util.c

```diff
@@ -193,6 +195,7 @@ const struct req_str_id attr_str_id_table[] = {
 	{  "type",              LDMSD_ATTR_TYPE  },
 	{  "udata",             LDMSD_ATTR_UDATA  },
 	{  "uid",               LDMSD_ATTR_UID  },
+	{  "update_interval",  LDMSD_ATTR_INTERVAL  },
```

Comment:
Just a nit. Could you please fix the text alignment to match the surrounding text?

Reply:
> Just a nit. Could you please fix the text alignment to match the surrounding text?

Done